app/controllers/comments_controller.rb

```diff
@@ -4,7 +4,7 @@ def create
     @comment = @user.comments.new(comment_params)
 
     if @comment.save
-      redirect_to root_path
+      redirect_to root_path, notice: 'Comment was successfully created'
     else
       render :new
```

Comment:
- [ ] Kindly would you please render a flash message when the comment is not saved. This will help the user to know the reason his comment was not saved.

Reply:
Of course! That's a great suggestion :+1: